aws/instrumentation_test.go

```diff
@@ -176,6 +208,14 @@ func wrapIntoSQSMessage(testMsg []byte, receiptHandle *string, md map[string]str
 	}, nil
 }
 
+func mustEncodeFilterPolicy(filter map[string]string) *string {
+	f, e := encodeFilterPolicy(filter)
+	if e != nil {
+		panic(e)
```

Comment:
Huh? Why are we panicking? If it's an error, return the error and fail the test.

Reply:
yet another pattern I found in the wild... I found it in the AWS API and in gotmpl. It's a test helper func so you can inline a filter policy in test functions. Not intended for use outside of test files

Reply:
But do we panic an abort the tests? Or does the test framework catch it and just log it as a test failure?

Reply:
great point -- I hadn't thought about it like that. I just pushed a change to make them call `t.Fatalf` instead of panic, which is the right way to go about it.